IDE/Espressif/ESP-IDF/examples/wolfssl_server/main/wifi_connect.c

```diff
@@ -128,7 +137,52 @@ static esp_err_t wifi_event_handler(void *ctx, system_event_t *event)
 /* entry point */
 void app_main(void)
 {
-    ESP_LOGI(TAG, "Start app_main...");
+    ESP_LOGI(TAG, "--------------------------------------------------------");
+    ESP_LOGI(TAG, "--------------------------------------------------------");
+    ESP_LOGI(TAG, "---------------------- BEGIN MAIN ----------------------");
+    ESP_LOGI(TAG, "--------------------------------------------------------");
+    ESP_LOGI(TAG, "--------------------------------------------------------");
+    ESP_LOGI(TAG, "CONFIG_IDF_TARGET = %s", CONFIG_IDF_TARGET);
+    ESP_LOGI(TAG, "LIBWOLFSSL_VERSION_STRING = %s", LIBWOLFSSL_VERSION_STRING);
+
+#if defined(WOLFSSL_MULTI_INSTALL_WARNING)
+    ESP_LOGI(TAG, "");
+    ESP_LOGI(TAG, "WARNING: Multiple wolfSSL installs found.");
```

Comment:
This seems like something that should be a compile time warning. Not runtime.

Reply:
Well, it's both: as the make/build process is rather long and the build warning could be missed.

It would be good to include the app as well for higher visibility.